openslides/utils/rest_api.py

```diff
@@ -30,6 +32,69 @@
 router = DefaultRouter()
 
 
+class IdManyRelatedField(ManyRelatedField):
+    field_name_suffix = '_id'
+
+    def bind(self, field_name, parent):
+        self.source = field_name[:-len(self.field_name_suffix)]
+        super().bind(field_name, parent)
+
+
+class IdPrimaryKeyRelatedField(PrimaryKeyRelatedField):
+    """
+    Field, that renames the field name to FIELD_NAME_id.
+
+    Only works together the our ModelSerializer.
+    """
+    field_name_suffix = '_id'
+
+    def bind(self, field_name, parent):
+        """
+        Called when the field is bound to the serializer.
+
+        Changes the source  so that the original field name is used (removes
+        the _id suffix).
+        """
+        if field_name:
+            self.source = field_name[:-len(self.field_name_suffix)]
```

Comment:
Can you add a comment or docstring like this:
`In case of many=True relations this class is used child_relation class, see first line in many_init() classmethod below. In this case the field_name is an empty string which of course cannot be shortened.`